x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/SecurityRealmSettingsTests.java

```diff
@@ -52,8 +53,11 @@ protected Settings nodeSettings(int nodeOrdinal) {
             final Path jwkSet = createTempFile("jwkset", "json");
             OpenIdConnectTestCase.writeJwkSetToFile(jwkSet);
 
+            MockSecureSettings mockSecureSettings =
```

Comment:
This fails because the truststore password is missing, but I can't figure out why. I'll check again my morning, but @tvernum if you have any ideas - I'm all ears